### PR TITLE
Update Set-CsTeamsAppSetupPolicy.md

### PR DESCRIPTION
**Pull request by xuanzhou4**
Add some usage examples and parameter descriptions.

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Review by get-itips — COMMENTED** [review]
Can we do the same for new-csteamsappsetuppolicy ? Thanks

**Review by dariomws — APPROVED** [review]

**Comment by xuanzhou4** [comment]
> Can we do the same for new-csteamsappsetuppolicy ? Thanks

yes,will do it.

**Comment by yogkumgit** [comment]
@JohanFreelancer9

File: skype/skype-ps/skype/Set-CsTeamsAppSetupPolicy.md [hunk 1]
```diff
@@ -44,7 +44,56 @@ Apps are pinned to the app bar. This is the bar on the side of the Teams desktop
 
 ### Example 1
 
-Intentionally not provided.
+```powershell
+$identity = "test"
+# Create new teams app setup policy named "Set-Test".
+New-CsTeamsAppSetupPolicy -Identity Set-$identity
```
[the remark this answers is not in the document]
Comment (xuanzhou4) [reply skype/skype-ps/skype/Set-CsTeamsAppSetupPolicy.md]: thanks for your suggestion,fixed it.

**Comment by yogkumgit** [comment]
@get-itips @tseward

File: skype/skype-ps/skype/Set-CsTeamsAppSetupPolicy.md [hunk 2]
```diff
@@ -94,7 +138,7 @@ Accept wildcard characters: False
 ```
 
 ### -Identity
-Do not use.
+Name of App setup policy. If empty, all Identities will be used by default.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Set-CsTeamsAppSetupPolicy.md:141]:
```suggestion
Name of app setup policy. If empty, all identities will be used by default.
```

File: skype/skype-ps/skype/Set-CsTeamsAppSetupPolicy.md [hunk 3]
```diff
@@ -123,11 +167,71 @@ Accept pipeline input: True (ByValue)
 Accept wildcard characters: False
 ```
 
+### -AllowUserPinning
+If you turn this on, the user’s existing app pins will be added to the list of pinned apps set in this policy. Users can rearrange, add, and remove pins as they choose. If you turn this off, the user’s existing app pins will be removed and replaced with the apps defined in this policy
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Set-CsTeamsAppSetupPolicy.md:171]:
```suggestion
If you turn this on, the user’s existing app pins will be added to the list of pinned apps set in this policy. Users can rearrange, add, and remove pins as they choose. If you turn this off, the user’s existing app pins will be removed and replaced with the apps defined in this policy.
```

File: skype/skype-ps/skype/Set-CsTeamsAppSetupPolicy.md [hunk 4]
```diff
@@ -123,11 +167,71 @@ Accept pipeline input: True (ByValue)
...
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
 ### -PinnedAppBarApps
-Do not use.
+Pinning an app displays the app in the app bar in Teams client. Admins can pin app and they can allow users to pin. Pinning is a used to highlight apps that are needed the most by users and promote ease of access.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Set-CsTeamsAppSetupPolicy.md:201]:
```suggestion
Pinning an app displays the app in the app bar in Teams client. Admins can pin apps and they can allow users to pin apps. Pinning is used to highlight apps that users need the most and promote ease of access.
```

File: skype/skype-ps/skype/Set-CsTeamsAppSetupPolicy.md [hunk 5]
```diff
@@ -123,11 +167,71 @@ Accept pipeline input: True (ByValue)
...
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -AppPresetList
+Choose which apps and messaging extensions you want installed in your users' personal Teams environment and in meetings they create. Users can install other available apps from the Teams app store.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Set-CsTeamsAppSetupPolicy.md:231]:
```suggestion
Choose which apps and messaging extensions you want to be installed in your users' personal Teams environment and in meetings they create. Users can install other available apps from the Teams app store. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @xuanzhou4, kindly commit the suggested changes. Many thanks.

**Review by get-itips — CHANGES_REQUESTED** [review]
Please see here as an example https://learn.microsoft.com/en-us/powershell/module/skype/set-csteamsmeetingpolicy?view=skype-ps how "Examples" should be formatted,
the description should be below the powershell code outside of it.

**Review by get-itips — CHANGES_REQUESTED** [review]
Some examples do not have description below the PowerShell

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@xuanzhou4

**Comment by yogkumgit** [comment]
@xuanzhou4 Gentle reminder

**Comment by yogkumgit** [comment]
@xuanzhou4 Is this PR still relevant, please implement he feedback else we may proceed to close it. Thanks

**Comment by xuanzhou4** [comment]
> @xuanzhou4 Is this PR still relevant, please implement he feedback else we may proceed to close it. Thanks

do you mean this feedback? "Some examples do not have description below the PowerShell" sorry I didn't notice this suggestion,will add some description under examples. Thanks!

**Comment by yogkumgit** [comment]
@get-itips @tseward

**Review by get-itips — APPROVED** [review]

**Comment by xuanzhou4** [comment]
> @xuanzhou4 Is this PR still relevant, please implement he feedback else we may proceed to close it. Thanks

Hi, I added descriptions under these examples, could you please help to merge this PR? thanks!

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**